commit 755cac44d9324857e04213cac47f4608131bed24
Author:     Keith Smiley <keithbsmiley@gmail.com>
AuthorDate: Thu Jan 15 20:58:46 2026 +0000
Commit:     Keith Smiley <keithbsmiley@gmail.com>
CommitDate: Thu Jan 15 20:58:46 2026 +0000

    Fix clang-tidy config symlinking
    
    Something about escaping this variable was wrong, but you only saw that
    if your config file wasn't already in the root of the repo.

Notes:
    Pile-URL: https://github.com/erenon/bazel_clang_tidy/pull/102

diff --git a/clang_tidy/clang_tidy_test.bzl b/clang_tidy/clang_tidy_test.bzl
index 2a21c15..da52e88 100644
--- a/clang_tidy/clang_tidy_test.bzl
+++ b/clang_tidy/clang_tidy_test.bzl
@@ -57,9 +57,8 @@ def _clang_tidy_rule_impl(ctx):
 set -euo pipefail
 
 readonly bin="{clang_tidy_bin}"
-readonly config="{clang_tidy_config}"
 
-test -e .clang-tidy || ln -s -f \\$config .clang-tidy
+test -e .clang-tidy || ln -s -f "{clang_tidy_config}" .clang-tidy
 if [[ ! -f .clang-tidy ]]; then
   echo "error: failed to setup config" >&2
   exit 1
